sktime/transformations/series/fourier.py

```diff
@@ -0,0 +1,202 @@
+# -*- coding: utf-8 -*-
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+"""Fourier features for time series with long/complex seasonality."""
+
+__author__ = ["ltsaprounis"]
+
+import warnings
+from copy import deepcopy
+from typing import List, Union
+
+import numpy as np
+import pandas as pd
+
+from sktime.transformations.base import BaseTransformer
+
+
+class FourierFeatures(BaseTransformer):
+    """Fourier Features for time series seasonality.
+
+    Fourier Series terms can be used as explanatory variables for the cases of multiple
+    seasonal periods and or complex / long seasonal periods [1][2]. For every seasonal
+    period, sp and fourier term k pair there are 2 fourier terms sin_sp_k and cos_sp_k:
+    - sin_sp_k = sin(2*pi*k*t/sp)
```

Comment:
Could you write out what precisely the series is this is transformed to?
Otherwise this might be very confusing.